react/components/Share/index.js

```diff
@@ -47,17 +64,17 @@ class Share extends Component {
           <circle
             cx="1em"
             {...loaderStyles}
-            {...loaderProps['vtex-share__button--loader-1']}
+            {...button1}
```

Comment:
I think that should be better to rename this variables as `firstButton`, `secondButton` and `thirdButton`.